larpandoracontent/LArObjects/LArTwoDSlidingFitResult.cc

```diff
@@ -572,7 +572,10 @@ void TwoDSlidingFitResult::CalculateAxes(const CartesianPointVector &coordinateV
     for (const CartesianVector &coordinate : coordinateVector)
         minProjection = std::min(minProjection, fitDirection.GetDotProduct(coordinate - centroid));
 
-    m_axisIntercept = centroid + (fitDirection * minProjection);
+    // Define layers based on centroid rather than individual extremal hits
+    const float fitProjection(layerPitch * std::floor(minProjection / layerPitch));
```

Comment:
Same comment as above, regarding minProjection still holding a float max value and then (potentially) being divided by a number less than one.